Taskfile.yml

```diff
@@ -68,8 +68,8 @@ tasks:
   format:
     desc: Format the code (and imports).
     cmds:
-      - isort odmantic tests
-      - black odmantic tests
+      - python -m isort odmantic tests
+      - python -m black odmantic tests
```

Comment:
I think we should now use ruff format instead. I forgot to edit this config when I migrated

Reply:
I just saw I haven't enabled it before. I just did in #425